src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -322,6 +322,8 @@
 
     <Field Name="Left" Type="BoundExpression"/>
     <Field Name="Right" Type="BoundExpression"/>
+    <Field Name="ConvertedLeft" Type="BoundExpression" SkipInVisitor="true"/>
```

Comment:
><Field Name="ConvertedLeft" Type="BoundExpression" SkipInVisitor="true"/> [](start = 4, length = 73)

It would be good to add a comment saying that these properties are meant for SemanticModel use only